src/guides/v2.3/extension-dev-guide/framework/array-manager.md

```diff
@@ -1,6 +1,8 @@
 ---
 group: php-developer-guide
 title: Array Manager
+contributor_name: Christophe Ferreboeuf
```

Comment:
Please remove the contributor items as this is not a new topic

Reply:
I spent 4h checking different scenarios as the documentation in the code itself is really sparse, I refactored most of the article as the examples were mostly copy-paste from existing code and I need to remove the bit of recognition this work can bring me. Sorry but it does not interest me. You can do whatever you want with this PR but I will not remove my contribution items.